### PR TITLE
release 2.5.0

### PR DESCRIPTION
**Pull request by root-ansh-ct**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,13 @@
 Change Log
 ==========
 
+Version 2.5.0 *(May 31, 2022)*
+-------------------------------------------
+- Supports [CleverTap Android SDK v4.4.0](https://github.com/CleverTap/clevertap-android-sdk/releases/tag/core-v4.4.0)
+- Supports [CleverTap iOS SDK v3.10.0](https://github.com/CleverTap/clevertap-ios-sdk/releases/tag/3.10.0)
+
```
Comment (root-ansh-ct) [inline CHANGELOG.md:8]: verify support for android 12 by running the sample on an emulator and mention it here too

File: Samples/Cordova/ExampleProject/platforms/android/app/src/main/assets/www/js/index.js [hunk 2]
```diff
@@ -167,104 +157,105 @@ onDeviceReady: function() {
     //      }
```
Comment (root-ansh-ct) [inline Samples/Cordova/ExampleProject/platforms/android/app/src/main/assets/www/js/index.js:157]: enhance ui by creating seperate callers for each function (similar to native sdk sample)

File: plugin.xml [hunk 3]
```diff
@@ -100,15 +100,15 @@
...
+        <framework src="com.github.bumptech.glide:glide:4.12.0"/>
+        <framework src="androidx.appcompat:appcompat:1.3.1"/>
         <framework src="androidx.core:core:1.3.0" />
-        <framework src="androidx.fragment:fragment:1.1.0" />
-        <framework src="androidx.recyclerview:recyclerview:1.1.0" />
+        <framework src="androidx.fragment:fragment:1.3.6"/>
+        <framework src="androidx.recyclerview:recyclerview:1.2.1"/>
         <framework src="androidx.viewpager:viewpager:1.0.0" />
-        <framework src="com.google.android.material:material:1.2.1" />
-        <framework src="com.android.installreferrer:installreferrer:2.1" />
```
Comment (root-ansh-ct) [inline plugin.xml:111]: ensure these versions match the one in core sdk version.properties

File: plugin.xml [hunk 4]
```diff
@@ -100,15 +100,15 @@
...
+        <framework src="com.google.android.material:material:1.4.0"/>
+        <framework src="com.android.installreferrer:installreferrer:2.2"/>
 
         <hook type="after_plugin_add" src="scripts/androidAfterPluginAdd.js" />
```
Comment (root-ansh-ct) [inline plugin.xml:114]: check for support of fcm push ?

check for android 12 working without push templates?

check for app working when push templates is added directly in android/ build.gradle or properties file

Comment (root-ansh-ct) [reply plugin.xml:114]: where to call `CleverTapAPI.setNotificationHandler((NotificationHandler)new PushTemplateNotificationHandler());`?

**Review by piyush-kukadiya — APPROVED** [review]
LGTM

**Pull request merged**